src/lib/radio/_radio-theme.scss

```diff
@@ -53,6 +39,22 @@
     &.mat-warn {
       @include _mat-radio-color($warn);
     }
+
+    &.mat-radio-disabled {
+      &.mat-radio-checked .mat-radio-outer-circle,
```

Comment:
Is that just too make sure that the specificity is high enough? Since you already moved the code below it should be already enough?